### ISSUE TITLE
Review - Hyperparameters

### ISSUE THREAD

**Issue by jwagemann** [esowc/ml_fire#9]
- Please provide a summary table of used hyperparameters: kernel function and parameters/ soft-margin constant C, loss function, etc. 

- If you plan to use NN, please provide information on: activation function, loss function, learning rate, neurons in each layer, any hidden layers.

- Please comment on the above choices.
